supplier_app/views.py

```diff
@@ -121,4 +121,16 @@ def form_valid(self, forms):
 
 
 class ApTaxpayers(LoginRequiredMixin, IsApUser, TemplateView):
-    template_name = 'AP_app/ap-taxpayers.html'
\ No newline at end of file
+    model = TaxPayerArgentina
+    template_name = 'AP_app/ap-taxpayers.html'
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        context['taxpayers'] = self.get_queryset()
+        return context
+
+    def get_queryset(self):
+        queryset = \
+            TaxPayerArgentina.objects.filter(taxpayer_state='PENDING') | \
+            TaxPayerArgentina.objects.filter(taxpayer_state='CHANGE REQUIRED')
```

Comment:
Instead of this 
```
queryset = TaxPayerArgentina.objects.filter(taxpayer_state='PENDING') | \
            TaxPayerArgentina.objects.filter(taxpayer_state='CHANGE REQUIRED')
``` 
You should find out if there is a simpler way with django, something like:
```queryset = TaxPayerArgentina.objects.filter(taxpayer_state__in=['PENDING', 'CHANGE REQUIRED'])```